ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/shared/header/header.component.spec.ts

```diff
@@ -1,64 +1,50 @@
-import { async, ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
-
-import { HeaderComponent } from './header.component';
-import { Router } from '@angular/router';
-import { ProfileService } from 'src/app/services/profile.service';
-import { UserProfile } from 'src/app/modules/shared/models/user-profile.model';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
 import { By } from '@angular/platform-browser';
-
+import { RouterTestingModule } from '@angular/router/testing';
+import { UserProfile } from 'src/app/modules/shared/models/user-profile.model';
+import { ProfileService } from 'src/app/services/profile.service';
+import { HeaderComponent } from './header.component';
 
 export class MockProfileService {
-
-  getUserProfile(): Promise<UserProfile> {
-      const profile = new UserProfile();
-      profile.role = 'professional';
-      profile.email = 'professional@hearings.hmcts.net';
-      return Promise.resolve(profile);
-  }
-
-  public get isLoggedIn(): boolean {
-      return true;
-  }
+    getUserProfile(): Promise<UserProfile> {
+        const profile = new UserProfile();
+        profile.role = 'professional';
+        profile.email = 'professional@hearings.hmcts.net';
```

Comment:
@hearings.hmcts.net email accounts might be flagged by the security team, can we just use @hmcts.net instead?